modular_pariah/modules/radiosound/code/headset.dm

```diff
@@ -0,0 +1,13 @@
+/obj/item/radio/headset
+	var/radiosound = 'modular_pariah/modules/radiosound/sound/radio/common.ogg'
+
+/obj/item/radio/headset/syndicate //disguised to look like a normal headset for stealth ops
+	radiosound = 'modular_pariah/modules/radiosound/sound/radio/syndie.ogg'
+
+/obj/item/radio/headset/headset_sec
+	radiosound = 'modular_pariah/modules/radiosound/sound/radio/security.ogg'
+
+/obj/item/radio/headset/talk_into(mob/living/M, message, channel, list/spans, datum/language/language, list/message_mods, direct = TRUE)
```

Comment:
*(Both of these are already like this on Skyrat, but may as well do a bit of tidying.)*
The `direct` argument never comes up anywhere other than here, so it can be removed:
```suggestion
/obj/item/radio/headset/talk_into(mob/living/M, message, channel, list/spans, datum/language/language, list/message_mods)
```